Gemfile

```diff
@@ -1,4 +1,9 @@
 source 'https://rubygems.org'
 
-# Specify your gem's dependencies in vagrant-ovirt.gemspec
-gemspec
+group :development do
+  gem 'vagrant', :git => 'git://github.com/mitchellh/vagrant.git'
+end
+
+group :plugins do
+  gem 'vagrant-ovirt3', :path => '.'
```

Comment:
As far as I understand gemspec defines a :default group for gems with no explicit group and :development group for development dependencies.

The change though switches the scope to vagrant gem source and adds the current source tree under plugins group (with all dependencies) which makes it possible to use it as a vagrant plugin through bundler.